### ISSUE TITLE
Intern interview format

### ISSUE THREAD

**Issue by sagesharp** [sagesharp/outreachy-django-wagtail#197]
One idea for promoting Outreachy is to surface the stories and experiences of current and past Outreachy interns. We could turn the interviews into a series of blog posts that we can use during an Outreachy fundraiser. Ideally, this would be tied to a longitudinal survey to see where the Outreachy alums are, and whether the program helped them with their career or volunteer work.

I'm using this issue to generate question ideas, and document any past articles/interviews we've had with Outreachy interns.

**Comment by sagesharp** [comment]
@shraddhabarke Sorry for taking so long to get back to this! I think you're on the right track here.

I've added a couple more questions, and regrouped yours so that they're in chronological order:

 1. Did you use any Free and Open Source Software (FOSS) before your internship with Outreachy?
1. Did you contribute to FOSS before your Outreachy internship?
1. How did you find out about the Outreachy internships?
1. What were your first thoughts about applying? Were you intimidated, confident, or worried?
1. What was your Outreachy application experience like? What advice, if any, would you give to the future Outreachy applicants?
1. How was your experience with the FOSS community during your internship?
1. What new skills did you learn while working on the project? The skills could be technical or non-technical skills related to collaboration, communication, writing, graphical design, translation, or anything you learned during your internship.
1. Did you feel the motivation to continue contributing after the internship period? Are you still involved with the community?
1. Did your Outreachy internship give you more confidence to collaborate with other FOSS communities? Did the internship increase your passion about FOSS?
1. Did you use the Outreachy travel fund to attend a conference or event? If so, did that event help you learn new skills or expand your network? What was your experience at that event?
1. What happened after your Outreachy internship ended? Did you finish your studies, find a full-time job, or decide to do something new?
1. Do you use any of the skills you learned during your internship in your current job, studies, or hobbies?
1. Has the Outreachy internship contributed positively towards your career? If so, how?
1. Does your job or university use FOSS? Do you contribute to FOSS as part of your classes, research, or job?

If you have time, blog posts in a "story" format tend to create a better connection to the audience that blog posts in a "question and answer" format. Here's two examples:

 - [Question and answer format](https://opensource.com/business/13/11/linux-kernel-interns-outreach-women)
 - [Story format](https://medium.freecodecamp.org/how-i-beat-the-odds-and-became-an-outreachy-intern-9a92f47cb44e?gi=a3361ccf4a17)

The first article feels choppy, with no chronological narrative or common theme among the questions. There's some interesting information, but the audience really has to hunt for meaning.

The second article has an emotional hook that grabs the audience, and a clear chronological narrative, building up the audience to root for this person to get accepted for Outreachy. It's a bit lengthy with the technical details, and I think could use a bit more human touch/impact at the end. However, I think we can replicate the same sort of story format by talking about the person's life before Outreachy, what they learned during Outreachy, and then how Outreachy helped them in the next step in their career or studies.

It requires a bit more back-and-forth and you might have to ask interns additional follow-up questions. You might get more of a story from doing a video conference with an alumni and recording the meeting, since you can get more of a back and forth with them.

Let me know what you think.

**Comment by shraddhabarke** [comment]
Thanks a lot for the questions! Yes indeed, I intended to write the posts in a story format to make it more appealing. Regarding the choice of alumni, I thought it would be best to increase diversity by choosing interns from different organizations, geographical locations and at various stages in their careers. Does that sound good?

Also, I think medium would be a great place to put up these articles! :)

**Comment by sagesharp** [comment]
@shraddhabarke Yes - I agree we should have a good mix of people. I would also like to highlight some folks who are especially under-represented in tech, like African American women and trans people.

I don't want to publish this on medium because medium recently switched to monetizing the content people post on it. We have a blog set up on Outreachy (although not well publicized, and doesn't have an RSS feed). I can set you up with permission to draft blog posts. I'll send you an email with details on how to log in and make draft blog posts.

**Comment by shraddhabarke** [comment]
Okay sure, but since the point is to increase awareness about Outreachy, I think it's best to put them up where a majority of people can easily access. :) ( doesn't have to be Medium )

**Comment by sagesharp** [comment]
We're planning to promote them on Twitter and link the blog posts from the tweets.

**Comment by shraddhabarke** [comment]
That sounds awesome!

**Comment by nerdyvaishali** [comment]
I just happen to come across this while looking at some code on this repo so I don't know if there is already any other issue regarding the same or not but I think one other way to outreach more about the program is via videos. Probably people can choose what would be their preferred way of sharing a story but we can definitely give them an idea. For example, giving above questions and asking if they would like to share it via blog post or video. 

Also, if planned in advance or informed in advance, I'm sure people can get more ideas or plans to do the same. Like I know so many people in Bangalore who have been part of Outreachy [applicants or interns] and it has affected in many good ways to their careers. So, I can definitely try to plan a meetup sort of thing where they talk about their stories etc. Sorry for throwing so many ideas. I'm just usually so excited for stuff like this :)
